src/lib/ngxs-firestore.service.ts

```diff
@@ -61,7 +61,7 @@ export abstract class NgxsFirestore<T> {
     return this.page$(queryFn).pipe(take(1));
   }
 
-  public createId() {
+  public createId(value: Partial<T>) {
```

Comment:
do we need to implement this at some point?

Reply:
This is available for the purpose of making the object available to anyone wanting to overload the function and build the id from data in the object. The default implementation would not use this parameter because it just gets the id from firebase.

Reply:
shouldnt we make it optional?

Reply:
After our discussion, it is better to not have the parameter, but to rely on the caller to set a custom id if needed.